src/Components/TimeSignature/timeSignature.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+
+import styled from "styled-components";
+
+const TimeSigStyle = styled.div`
+  display: flex;
+  flex-direction: row;
+  input {
+    width: 25px;
```

Comment:
For our project we will be using `rem` instead of `px`. Would you be able to convert all the pixel uses to rem? This can be done by deviding the number by 16. For example `25px` would be `1.5625rem`.

- [ ] Convert `px` to `rem`